osm2gtfs/core/osm_connector.py

```diff
@@ -514,21 +601,22 @@ def _generate_shape(self, route_variant, query_result_set):
     def _is_valid_stop_candidate(self, stop):
         """Helper function to check if a stop candidate has a valid tagging
 
-        Returns True or False
+        :return bool: Returns True or False
 
         """
+        valid = False
         if 'public_transport' in stop.tags:
             if stop.tags['public_transport'] == 'platform':
```

Comment:
Why don't we check for `stop_position`?

Reply:
No, but this is not the scope of this PR. It is how it is currently implemented (in master).

Reply:
This is treated in #100.